backend/tests/database/test_client.py

```
@@ -0,0 +1,209 @@
+import pytest
+from database._client import get_firestore, get_async_firestore, transaction
+from unittest.mock import patch, MagicMock, call
+from google.cloud import firestore
```

Ruff

`google.cloud.firestore` imported but unused

Remove unused import: `google.cloud.firestore`

(F401)